### PR TITLE
README Updates

### PR DESCRIPTION
**Pull request by gilbode**
Add CONTRIBUTING.md
Updates to README to largely delegate to other docs for content

### CODE REVIEW

**Review by markxnelson — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -14,62 +11,41 @@ Verrazzano includes the following capabilities:
...
-
-See the `./operator/scripts/install/chart/values.yaml` file for the full list of configurable parameters that can be set using
-`--set parameter=value` when installing the Helm chart.
-
-
-## More information
-
-For more details, see the [installation instructions](install.md).
+If you want to build and install Verrazzano from this repository, follow the instructions in the [operator](./operator) directory.  For
+more detailed installation instructions please see the [Verrazzano documentation](https://verrazzano.io/docs/).
```
Comment (rosemarymarano) [inline README.md:51]: For more detailed installation instructions please see -> For more detailed installation instructions, see
see the [Verrazzano documentation](https://verrazzano.io/docs/) -> see the [Verrazzano Installation Guide](https://verrazzano.io/docs/setup/install/installation/).

File: README.md [hunk 2]
```diff
@@ -14,62 +11,41 @@ Verrazzano includes the following capabilities:
 - Integrated security
 - DevOps and GitOps enablement
 
-This repository contains a Kubernetes operator for installing Verrazzano and example applications for use with Verrazzano.
+This repository contains the following content:
 
-> **NOTE**: This alpha release of Verrazzano is intended for installation in a single OKE or Oracle Linux Cloud Native Environment (OLCNE) cluster. You should install Verrazzano only in a cluster that can be safely deleted when your evaluation is complete.
+  - [Verrazzano platform operator](./operator) - a [Kubernetes operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) that can 
+    be deployed to a Verrazzano cluster and can install Verrazzano components into and uninstall those components from the cluster
```
Comment (rosemarymarano) [inline README.md:17]: that can be deployed to a Verrazzano cluster and can install Verrazzano components -> that can be deployed to a Verrazzano cluster and install Verrazzano components

File: README.md [hunk 3]
```diff
@@ -14,62 +11,41 @@ Verrazzano includes the following capabilities:
...
+    which the operator is deployed in.
```
Comment (rosemarymarano) [inline README.md:18]: cluster which the operator is deployed in -> cluster in which the operator is deployed.

File: README.md [hunk 4]
```diff
@@ -14,62 +11,41 @@ Verrazzano includes the following capabilities:
...
-5. Run the following commands in the OCI Cloud Shell:
 
-```
-    export KUBECONFIG=~/.kube/config
-    kubectl apply -f operator/deploy/operator.yaml
-    kubectl -n verrazzano-install rollout status deployment/verrazzano-platform-operator
-    kubectl apply -f operator/config/samples/install-default.yaml
-    kubectl wait --timeout=20m --for=condition=InstallComplete verrazzano/my-verrazzano
-```
+For detailed documentation on using Verrazzano please see the [Verrazzano documentation](https://verrazzano.io/docs/).
```
Comment (rosemarymarano) [inline README.md:23]: For detailed documentation on using Verrazzano please see the -> For detailed documentation on using Verrazzano, see the

**Review by rosemarymarano — COMMENTED** [review]
Some edits/suggestions

**Comment by gilbode** [comment]
@rosemarymarano i think i addressed your comments

File: README.md [hunk 5]
```diff
@@ -14,62 +11,41 @@ Verrazzano includes the following capabilities:
 - Integrated security
 - DevOps and GitOps enablement
 
-This repository contains a Kubernetes operator for installing Verrazzano and example applications for use with Verrazzano.
+This repository contains the following content:
 
-> **NOTE**: This alpha release of Verrazzano is intended for installation in a single OKE or Oracle Linux Cloud Native Environment (OLCNE) cluster. You should install Verrazzano only in a cluster that can be safely deleted when your evaluation is complete.
+  - [Verrazzano platform operator](./operator) - a [Kubernetes operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) that can 
+    be deployed to a Verrazzano cluster and install Verrazzano components into and uninstall those components from the cluster in
+    which the operator is deployed in.
```
Comment (rosemarymarano) [inline README.md:18]: in which the operator is deployed in. -> in which the operator is deployed.

**Review by rosemarymarano — APPROVED** [review]
Just one small nit left; otherwise fine.

File: README.md (hunk 5 above)
Comment (gilbode) [reply README.md]: Done

**Pull request merged**